CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* Fix implementation of `LinearSplit` hint.
+
```

Comment:
I wouldn't include this in the Changelog. As the code that was fixed was added in the same release the fix is "invisible" to those updating cairo-rs to a newer version.